src/Symfony/Bundle/SecurityBundle/Command/UserPasswordEncoderCommand.php

```diff
@@ -0,0 +1,225 @@
+            ->addRow(array('Encoded password', $encodedPassword))
+        ;
+
+        $table->render();
+    }
+
+    /**
+     * Create the password question to ask the user for the password to be encoded.
+     *
+     * @param InputInterface  $input
+     * @param OutputInterface $output
+     *
+     * @return Question
+     */
+    private function createPasswordQuestion(InputInterface $input, OutputInterface $output)
+    {
+        $passwordQuestion = new Question("\n > <question>Type in your password to be encoded:</question> ");
+
+        $passwordQuestion->setValidator(function ($value) {
+            if ('' === trim($value)) {
+                throw new \Exception('The password must not be empty.');
+            }
+
+            return $value;
+        });
+        $passwordQuestion->setHidden(true);
+        $passwordQuestion->setMaxAttempts(20);
+
+        return $passwordQuestion;
+    }
+
+    /**
+     * Create the question that asks for the salt to perform the encoding.
+     * If there is no provided salt, a random one is automatically generated.
+     *
+     * @param InputInterface  $input
+     * @param OutputInterface $output
+     *
+     * @return Question
+     */
+    private function createSaltQuestion(InputInterface $input, OutputInterface $output)
+    {
+        $saltQuestion = new Question("\n > (Optional) <question>Provide a salt (press <enter> to generate one):</question> ");
+
+        $container = $this->getContainer();
+        $saltQuestion->setValidator(function ($value) use ($output, $container) {
+            if ('' === trim($value)) {
+                $value = hash('sha512', $container->get('security.secure_random')->nextBytes(30));
+
+                $output->writeln("\n<comment>The salt has been generated: </comment>".$value);
+                $output->writeln(sprintf("<comment>Make sure that your salt storage field fits this salt length: %s chars.</comment>\n", strlen($value)));
+            }
+
+            return $value;
+        });
+
+        return $saltQuestion;
+    }
+
+    /**
+     * Create the question that asks for the configured user class.
+     *
+     * @param InputInterface  $input
+     * @param OutputInterface $output
+     *
+     * @return Question
+     */
+    private function createUserClassQuestion(InputInterface $input, OutputInterface $output)
+    {
+        $userClassQuestion = new Question(" > <question>Provide your configured user class:</question> ");
+        $userClassQuestion->setAutocompleterValues(array('Symfony\Component\Security\Core\User\User'));
+
+        $userClassQuestion->setValidator(function ($value) use ($output) {
+            if ('' === trim($value)) {
+                $value = 'Symfony\Component\Security\Core\User\User';
+                $output->writeln("<info>You did not provide any user class.</info> <comment>The user class used is: Symfony\Component\Security\Core\User\User</comment> \n");
```

Comment:
Yes, that is what I mean. Since you configure `Symfony\Component\Security\Core\User\User` in the `setAutocompleterValues` and you set the `$value` to `Symfony\Component\Security\Core\User\User` if null, I guess you can configure the default value of the `user-class`.